musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/ToastDisplayer.java

```diff
@@ -0,0 +1,46 @@
+package org.catrobat.musicdroid.pocketmusic;
+
+import android.content.Context;
+import android.widget.Toast;
+
+import org.catrobat.musicdroid.pocketmusic.note.midi.MidiPlayer;
+
+/**
+ * Class to manage toasts shown while playing/stopping or when the track is done playing
+ */
+public class ToastDisplayer {
+    public static final String STARTED_PLAYING = "Playing";
+    public static final String STOPPED_PLAYING = "Stopped";
+    public static final String FINISHED_PLAYING = "Done";
```

Comment:
Strings should be stored in the strings.xml and the now unused String in strings.xml action_play_midi_dialog_title should be removed